tests/cicd/main_build.yml

```diff
@@ -351,3 +351,26 @@ jobs:
       pytest -s tests/cicd/src/notebook_integration_tests.py \
         --nbname examples/interpretation/segyconverter/02_segy_convert_sample.ipynb \
         --cwd examples/interpretation/segyconverter
+
+
+###################################################################################################
+# Stage 5: Docker tests
+###################################################################################################
+
+
+- job: docker_build_test
+  dependsOn: segyconverter_notebooks
+  timeoutInMinutes: 30
+  displayName: Docker build test
+  pool:
+    name: deepseismicagentpool
+  steps:
+  - bash: |
+      set -e
+      echo "build docker"
```

Comment:
need to add a way to remove old Docker image so that the new build starts from scratch and doesn't re-use layers from the previous Docker build